app/src/main/java/org/fossasia/openevent/general/speakercall/form/SessionIdentifier.kt

```diff
@@ -0,0 +1,12 @@
+package org.fossasia.openevent.general.speakercall.form
+
+object SessionIdentifier {
```

Comment:
Can it be declared as enum instead of an object?

Reply:
I don't know. I add the object above as string in order to match the session identifier (also string) from the server. I'm not sure how to do that with enum

Reply:
Screenshot already added above